templates/docs/examples/patterns/buttons/dark.html

```diff
@@ -3,12 +3,13 @@
 
 {% block standalone_css %}patterns_buttons{% endblock %}
 
+{% set is_dark = True %}
 {% block content %}
-<div class="p-strip is-dark is-shallow" style="background: #111">
-    <button class="p-button is-dark">Default button</button>
-    <button class="p-button--base is-dark">Base button</button>
-    <button class="p-button--positive is-dark">Positive button</button>
-    <button class="p-button--negative is-dark">Negative button</button>
-    <button class="p-button--brand is-dark">Brand button</button>
+<div class="p-strip is-shallow">
+    <button class="p-button">Default button</button>
```

Comment:
I'm not sure if that works, because buttons have not been migrated to new theming yet, so they need to have `is-dark` class on them (they don't inherit the theme).

You can keep the `is_dark = True` part to have class on body, but the `is-dark` class on buttons needs to stay for now. I know that your toggle will not work properly here, but it's fine.
(I commented in MM about the conditional, so that's another option as well).

Overall, the `is-dark` classes on buttons need to stay until buttons are properly themed.